src/Microsoft.OpenApi.Hidi/Microsoft.OpenApi.Hidi.csproj

```diff
@@ -10,6 +10,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.OData.Edm" Version="7.9.4" />
+    <PackageReference Include="Microsoft.OpenApi.OData" Version="1.0.8" />
```

Comment:
1.0.9 is now available.

Reply:
I downgraded to version 1.0.8 because I was running out of memory while trying to convert the clean metadata with descriptions from the metadata repo. Has this issue been fixed in the library?